app/models/affiliate.rb

```diff
@@ -726,4 +730,10 @@ def validate_managed_no_results_pages_guidance_text
 
     errors.add(:base, 'Additional guidance text is required when links are present.')
   end
+
+  def set_show_search_filter_settings
```

Comment:
This method ensures that search filter settings are disabled when the search engine is set to Bing.

Reply:
If we want to avoid accidentally setting it `true` on affiliates that are using bing maybe we can add a validation that ensures show_search_filter_settings can't be true if they're using bing

Reply:
This method is meant to ensure that the new parameter is always set to `false` when switching from SearchGov to Bing multiple times.

Reply:
I didn't fully explained my self. This can be better accomplished through a validation as it provides user feedback making clients aware of show_search_filter_settings needing to be turned off if there is a switch. What do you think?

Something like:

```ruby
  validates : show_search_filter_settings, absence: { message: "should be false when search_engine is bing7" }, if: :bing7_search_engine?

  def bing7_search_engine?
    search_engine == 'bing7'
  end
```

Reply:
Thanks, I like this approach because it’s more efficient than mine. I’ll update the PR shortly.